gradle/dependencies.gradle

```diff
@@ -79,6 +79,7 @@ versions += [
   kafka_11: "1.1.1",
   kafka_20: "2.0.1",
   kafka_21: "2.1.0",
```

Comment:
Should that be `2.1.1`?
I am asking because this list enumerates only the most recent bugfix releases and you removed `2.1.0` in `tests/docker/Dockerfile`.

Reply:
This is updated in #6596 -- separate PR to allow cherry-picking to older branches.